File: adjacentElementsProduct.js
function solution(inputArray) {
    let largest = -Number.MAX_VALUE;
    
    for (let i = 0; i < inputArray.length - 1; i++){
        let product = inputArray[i] * inputArray[i+1];
        
        if (product > largest){
            largest = product;
        }
    }
    
    return largest;
}

File: adjacentElementsProduct.cs
int solution(int[] inputArray) {
    int largestProduct = int.MinValue;
    
    for (int i = 0; i < inputArray.Length - 1; i++)
    {
        int workingProduct = inputArray[i] * inputArray[i + 1];
        if (workingProduct > largestProduct)
        {
            largestProduct = workingProduct;
        }
    }
    
    return largestProduct;
}
